1.download-data/downloadData.py

```diff
@@ -0,0 +1,23 @@
+from scripts import downloadData
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument(
+    "--downloadLocation",
+    type=str,
+    default="data",
+    dest="downloadLocation",
+    help="The account name of the blobStorage where the validation data is stored",
+)
+parser.add_argument(
+    "--downloadUrl",
+    type=str,
+    default=r"https://ndownloader.figshare.com/files/18501824?private_link=f41918598b1ff5116825",
+    dest="downloadUrl",
+    help="The account name of the blobStorage where the validation data is stored",
+)
```

Comment:
I think we should remove this argument.

The figshare links are static and this script will only need to be run once

Reply:
Also, the figshare link is now: https://doi.org/10.6084/m9.figshare.10247531.v1 (so no longer private)